msgraphforentra.json

```
@@ -0,0 +1,1008 @@
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "on poll",
+            "description": "Callback action for the on_poll ingest functionality",
+            "type": "ingest",
+            "identifier": "on_poll",
+            "read_only": true,
+            "parameters": {
+                "start_time": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 0
+                },
+                "end_time": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 1
+                },
+                "container_count": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored for schedule/interval polling only",
+                    "order": 2
+                },
+                "artifact_count": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 3
+                },
+                "container_id": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 4
+                }
+            },
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "dismiss users risk",
+            "description": "Dismiss the risk of one or more users",
+            "type": "generic",
+            "identifier": "dismiss_users_risk",
+            "read_only": false,
+            "parameters": {
+                "userids": {
+                    "data_type": "string",
+                    "description": "One or more userids to dismiss risk for",
+                    "order": 0
+                }
+            },
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list risk detections",
+            "description": "List risk detections",
+            "type": "investigate",
+            "identifier": "list_risk_detections",
+            "read_only": true,
+            "parameters": {
+                "limit": {
+                    "description": "Maximum number of risk detections to return (Defaults to 50)",
+                    "data_type": "numeric",
+                    "default": 50,
+                    "order": 0
+                },
+                "filter": {
+                    "description": "Filter risk detections based on property",
+                    "data_type": "string",
+                    "order": 2
+                },
+                "orderby": {
+                    "description": "Sort the risk detections based on property",
+                    "data_type": "string",
+                    "order": 3
```

Pretty sure static tests will get mad at this (I know, I think it's silly too)
```suggestion
                "filter": {
                    "description": "Filter risk detections based on property",
                    "data_type": "string",
                    "order": 1
                },
                "orderby": {
                    "description": "Sort the risk detections based on property",
                    "data_type": "string",
                    "order": 2
```

---

I actually prefer to use 10,20,30 etc, make's adding new params easier. But most of our apps don't do this, so I'll just accept it :)